### PR TITLE
Require the competitor to finish signing before the competitor may reset the timer

### PR DESCRIPTION
**Pull request by Rouxles**
Forum link: https://forum.worldcubeassociation.org/t/require-the-competitor-to-finish-signing-not-just-for-the-judge-to-finish-signing-before-the-competitor-may-reset-the-timer/1302

### CODE REVIEW

**Comment by Ivan-Brigidano** [comment]
I think it would be better to rephrase the Regulation A6f rather than adding a new Regulation.

**Comment by Rouxles** [comment]
> 
> 
> I think it would be better to rephrase the Regulation A6f rather than adding a new Regulation.

In the meeting minutes, was something on systematic leniency discussed for this or should I add an extra guideline/change it to A6f?
https://github.com/thewca/wca-regulations/pull/994/files: add a guideline or something on Systematic Leniency saying that this is not necessarily a DNF.

**Comment by Ivan-Brigidano** [comment]
Actually this was just an idea but I don't think we should add a Guideline saying that. It would be better to cover borderline cases with Systemic Leniency in my opinion.

What we do agreed is rewriting Regulation A6f rather than adding a new Regulation.

**Review by mirandapedro — APPROVED** [review]
LGTM!

I also agree very much with the thought of rewriting Regulations instead of adding new ones whenever possible. 

This change will surely be one that impacts competitors and causes incident the most, so we must be sure to include it in the quick summary of changes and highlight it to the Delegates so that they can implement this in their communities more smoothly.

File: wca-regulations.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
     - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).
     - A6e) After releasing the puzzle, the competitor must not touch or move the puzzle until the judge has inspected the puzzle. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
-    - A6f) The competitor must not reset the timer until the judge has recorded the result on the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
+    - A6f) The competitor must not reset the timer unless both the competitor and the judge have signed the attempt. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
    - A6f) The competitor must not reset the timer unless both the competitor and the judge have signed the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```

File: wca-regulations.md [hunk 2]
```diff
@@ -406,7 +406,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
     - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).
     - A6e) After releasing the puzzle, the competitor must not touch or move the puzzle until the judge has inspected the puzzle. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
-    - A6f) The competitor must not reset the timer until the judge has recorded the result on the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
+    - A6f) The competitor must not reset the timer unless both the competitor and the judge have signed the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```
Comment (Ivan-Brigidano) [inline wca-regulations.md]:
```suggestion
    - A6f) The competitor must not reset the timer until both the competitor and the judge have signed the score sheet. Penalty: disqualification of the attempt (DNF), at the discretion of the judge.
```

**Pull request merged**